pkgs/by-name/my/mygui/package.nix

```
@@ -63,10 +68,12 @@ stdenv.mkDerivation rec {
     "-DMYGUI_RENDERSYSTEM=${renderSystem}"
```

See:

- https://wiki.openmw.org/index.php?title=Development_Environment_Setup#MyGUI
- https://gitlab.com/OpenMW/openmw/-/issues/7717

```suggestion
    "-DMYGUI_RENDERSYSTEM=${renderSystem}"
    "-DMYGUI_DONT_USE_OBSOLETE=ON"
```